README.md

```diff
@@ -1,2 +1,27 @@
-# hra-ui
-Human Reference Atlas UIs Monorepo
+# Human Reference Atlas (HRA) and Functional Tissue Unit Explorer (FTU) - User Interfaces
+
+[![Tests]()](https://github.com/hubmapconsortium/hra-ui/actions/workflows/ci.yml)
```

Comment:
Please fix this so it shows the badges

Reply:
Where can I get the links to the badges for hra-ui?

Reply:
Follow the model from the thing you _literally_ copy and pasted from https://raw.githubusercontent.com/hubmapconsortium/ccf-ui/main/README.md